akka-distributed-data/src/main/protobuf/ReplicatedDataMessages.proto

```diff
@@ -72,47 +72,73 @@ message ORMap {
   }
   
```

Comment:
I have just added a grouping into `MapEntry`. This will allow to extend easier with coalesced deltas.